e2e/txserver/zeta_tx_server.go

```diff
@@ -289,20 +289,28 @@ func (zts ZetaTxServer) EnableHeaderVerification(account string, chainIDList []i
 // DeploySystemContractsAndZRC20 deploys the system contracts and ZRC20 contracts
 // returns the addresses of uniswap factory, router and erc20 zrc20
 func (zts ZetaTxServer) DeploySystemContractsAndZRC20(
-	account, erc20Addr string,
+	accountOperational, accountAdmin, erc20Addr string,
 ) (string, string, string, string, string, error) {
 	// retrieve account
```

Comment:
Would be really nice to return `(struct{}, error)` instead of 5 strings. e.g. `SystemDeployOutcome{}`